### ISSUE TITLE
ocw-www: about page

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#278]
As a user, I'd like to find out more about OCW 

#### Designs and Mockups

desktop: https://invis.io/N2121HAUUW3J#/461217843_About_Us
mobile: https://invis.io/FU1226TX74CV#/461387536_Mobile_-_About_Us_-_Subnav_-_Scroll 
mobile timeline: https://invis.io/65123QVGCNPG#/459413988_Mobile_Timeline

copy: https://docs.google.com/document/d/1NTIIJ_q0Vqyv9ZUFU66sVBM9n9EfJcTxct3mJMseDWs/edit

#### Acceptance Criteria:

- [ ] for now, this should all be part of the ocw-www theme. In the future, we may consider exposing some of it through studio. 
- [ ] As much as possible, styles should follow existing styles (e.g. fonts & colors)
- [ ] One long, scrolling page
- [ ] on-page nav should stay sticky, at top of viewport when user scrolls down (mobile UX TBD)
- [ ] clicking on item in on-page nav should smooth-scroll down to the appropriate section, not jump. 
- [ ] SRC images TBD. For now grab what you can from InVision 
- [ ] video at https://www.youtube.com/watch?v=0aAEamhJHUI 
- [ ] "Chalk Radio Podcast" can be an embed  from SimpleCast: `<iframe height="200px" width="100%" frameborder="no" scrolling="no" seamless src="https://player.simplecast.com/a9349695-05d2-4150-bba6-271cbf875ba0?dark=false"></iframe>`
- [ ] For the OCW Staff section, copy names, titles and photos from https://openlearning.mit.edu/our-team?term_node_tid_depth[72]=72
- [ ] Update the ocw-www home page to link to link "Learn More about the OCW Mission" to this page
- [ ] Update the site header (ocw-www and ocw-courses) to link "About OCW" to this page.
 
#### Out of Scope

- Omit "Educators Start Here" section and button until #issue is unblocked
- Omit "Find Courses with Instructor Insights" section & button until #issue is unblocked
- Omit "Open Textbooks" section & button until #279 is unblocked
- Omit "Open Learning Library" section & button until #280 unblocked

**Comment by gumaerc** [comment]
@pdpinch What is the expected URL here?  is `/about` okay or did we want `/about-us`?

**Comment by gumaerc** [comment]
@pdpinch The page contains several links in the "milestones" section.  I can assume where some of these links go but it would be best to have specific URLs for each link either here or in the design.

**Comment by pdpinch** [comment]
Best to keep it the same as the current site, /about

@pdpinch<https://github.com/pdpinch> What is the expected URL here? is /about okay or did we want /about-us?

**Comment by pdpinch** [comment]
Sorry, they should have been in the Google doc. But rather than delay things with busy work, you can scrape the URLs from https://ocw.mit.edu/about/milestones/

> @pdpinch <https://github.com/pdpinch> The page contains several links in the "milestones" section. I can assume where some of these links go but it would be best to have specific URLs for each link either here or in the design.

**Comment by gumaerc** [comment]
@pdpinch Thanks, I didn't notice they were in the Doc.  I should have checked there first.
